packages/web-react/src/components/Field/ValidationText.tsx

```diff
@@ -30,13 +32,15 @@ export const ValidationText = <T extends ElementType = 'div'>(props: ValidationT
 
   if (validationText) {
     return Array.isArray(validationText) ? (
-      <ul className={className} id={id}>
-        {validationText.map((item) => (
-          <li key={`validationText_${item}`}>{item}</li>
-        ))}
-      </ul>
+      <div className={className} id={id} role={role}>
```

Comment:
One thing to consider. I don't know if we should implement it. 

```suggestion
      <ElementTag className={className} id={id} role={role}>
```

🤷 

anyway, it will be always HTML invalid :D